pkg/lib/console.css

```diff
@@ -1,4 +1,4 @@
-@import "@xterm/xterm/css/xterm.css";
```

Comment:
This test fails here, `@use` doesn't exist in `.css`. I'll revert this one when tests are done
https://cockpit-logs.us-east-1.linodeobjects.com/pull-21764-a014eb74-20250325-121334-arch-other/log.html#133-2